processor-line.cpp

```diff
@@ -18,6 +18,5 @@ geometry_processor::wkbs_t
 processor_line::process_relation(osmium::Relation const &,
                                  osmium::memory::Buffer const &ways)
 {
-    // XXX are multilines really acceptable?
-    return m_builder.get_wkb_multiline(ways, false);
+    return m_builder.get_wkb_multiline(ways, 1000000);
```

Comment:
This was the value used before the regression from the libosmium change. I can add another 0 though.